packages/components/src/navigator/test/index.tsx

```diff
@@ -98,25 +105,6 @@ function CustomNavigatorButton( {
 	);
 }
 
-function CustomNavigatorButtonWithFocusRestoration( {
```

Comment:
Removed in favour of using `CustomNavigatorButton` everywhere, since today there are basically no differences between the two